scripts/lib.py

```diff
@@ -85,20 +85,6 @@ def get_env_var(var, debug_cmd = None):
   return val
 
 
-def get_seed(seed):
```

Comment:
note that Ibex sim.py uses this function, please make sure to update the Ibex script to match what is happening here.

Reply:
Good point. I'll add it to a vendoring patch today.